src/Sarif.Viewer.VisualStudio/Sarif.Viewer.VisualStudio.csproj

```diff
@@ -5,6 +5,8 @@
     <VSToolsPath Condition="'$(VSToolsPath)' == ''">$(MSBuildExtensionsPath32)\Microsoft\VisualStudio\v$(VisualStudioVersion)</VSToolsPath>
   </PropertyGroup>
   <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+  <Import Project="$(MSBuildProjectExtensionsPath)$(_TargetAssemblyProjectName)$(MSBuildProjectExtension).nuget.g.props"
```

Comment:
Hack: https://github.com/dotnet/Nerdbank.GitVersioning/issues/404

Reply:
Add this comment to the project file itself.

---
In reply to: [521584147](https://github.com/microsoft/sarif-visualstudio-extension/pull/267#discussion_r521584147) [](ancestors = 521584147)